### PR TITLE
Update downloadAndInsertOpenCV.sh

### PR DESCRIPTION
**Pull request by Rohan4045**

### CODE REVIEW

File: downloadAndInsertOpenCV.sh [hunk 1]
```diff
@@ -2,24 +2,24 @@
 
 # ios
 version=4.1.0
```
Comment (piotr-s-brainhub) [inline downloadAndInsertOpenCV.sh:4]: why don't you just change this version?

and why do you prefer an older version?

File: downloadAndInsertOpenCV.sh [hunk 2]
```diff
@@ -2,24 +2,24 @@
...
+base_url=https://razaoinfo.dl.sourceforge.net/project/opencvlibrary/${3.4.9}/
 
-wget ${base_url}/opencv-${version}-ios-framework.zip
-unzip -a opencv-${version}-ios-framework.zip
-cd ios
-cp -r ./../opencv2.framework ./
-cd ..
-rm -rf opencv-${version}-ios-framework.zip
-rm -rf opencv2.framework/
+# wget ${base_url}/opencv-${version}-ios-framework.zip
```
Comment (piotr-s-brainhub) [inline downloadAndInsertOpenCV.sh:7]: why did you comment out this code?

**Comment by piotr-s-brainhub** [comment]
@Rohan4045 

Are you going to reply?

Otherwise, I'm going to close this PR.

**Comment by Rohan4045** [comment]
I was trying to add opencv in my project, i have downloaded opencv, but not able to add, then i tried this script, and this also do the same work, . I have mistakenly commented out as i want only for Android. Sorry for that.

**Pull request closed without merging**